packages/financial-templates-lib/test/helpers/GasEstimator.js

```diff
@@ -47,7 +47,7 @@ contract("GasEstimator.js", function() {
         level: "info",
         transports: [new winston.transports.Console()]
       });
-      gasEstimator = new GasEstimator(dummyLogger, (updateThreshold = 2), (defaultFastPriceGwei = 10));
+      gasEstimator = new GasEstimator(dummyLogger, /* updateThreshold */ 2, /* defaultFastPriceGwei */ 10);
```

Comment:
What's going on here?

Reply:
Yeah @chrismaree, I'd like to get your eyes on this. It looks like this was using some pretty strange syntax to signal what variables were being set. It was, in effect, creating global variables with those names, but I think the intention was just to show the values that were being set in the constructor.

Reply:
I changed it to use comments to do that signaling, but if that wasn't the intention, we can swap it to some other syntax.

Reply:
ye, I think that was an error & your change makes sense. this just needs to signal that these are defaults in this test.